tests/e2e/standalone/init_run_custom_path_test.go

```diff
@@ -0,0 +1,205 @@
+//go:build e2e && !template
+// +build e2e,!template
+
+/*
+Copyright 2022 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package standalone_test
+
+import (
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"testing"
+
+	"github.com/dapr/cli/tests/e2e/common"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// TestStandaloneInitRunUninstallNonDefaultDaprPath covers init, version, run and uninstall with --runtime-path flag.
+func TestStandaloneInitRunUninstallNonDefaultDaprPath(t *testing.T) {
```

Comment:
Can we refactor the tests to not repeat certain common setup/cleanup actions and only focus on act/asserts? We can keep it for later as well, not a blocker.

Reply:
will be separate PR.